documentation/getting_started.md

```diff
@@ -243,6 +243,37 @@ WARNING: Default profile is used for sign. This signed package is valid for emul
 Package File Location: nnstreamer-example/Tizen.web/ImageClassificationOffloading/.buildResult/ImageClassificationOffloading.wgt
 ```
 
+We need a hybrid app to get the port value automatically from the offloading service.
+
+```bash
+$ cd nnstreamer-example/Tizen.web/ImageClassificationOffloadingService
+$ tizen build-native -a arm -c llvm -C Debug -- .
+[PLATFORM] Tizen-8.0
+...omitted...
+Total time: 00:00:00.314
+
+$ tizen package -t tpk -- Debug
+Initialize... OK
+...omitted...
+Package File Location: nnstreamer-example/Tizen.web/ImageClassificationOffloadingService/Debug/EQmf4iSfpX-1.0.0-arm.tpk
+
+$ cd nnstreamer-example/Tizen.web/ImageClassificationOffloading
+$ tizen package -t wgt -r ../ImageClassificationOffloadingService/Debug/EQmf4iSfpX-1.0.0-arm.tpk  -- .buildResult/ImageClassificationOffloading.wgt
+The active profile is used for signing. If you want to sign with other profile, please use '--sign' option.
+...omitted...
+Package( nnstreamer-example/Tizen.web/ImageClassificationOffloading/.buildResult/ImageClassificationOffloading.wgt ) is created successfully.
+```
+Tizen Web App build process is done automatically through github actions.
+So if you don't need to build it yourself, you can download it from this [link](https://github.com/nnstreamer/nnstreamer-example/actions/workflows/tizen-build.yml).
+
+Click `Run workflow` with main branch.
```

Comment:
https://github.com/nnstreamer/nnstreamer-example/actions/workflows/tizen-build.yml

Run workflow buttons from workflow_dispatch is only visiable to who has a permission now.

Reply:
Can we then make a pre-built package for this?

Reply:
Yes.
I just added workflow-dispatch option to the tizen-build workflow.
https://github.com/nnstreamer/nnstreamer-example/pull/358

A permission change is required to allow all users to trigger this workflow.

Reply:
@niley7464 https://github.com/orgs/nnstreamer/packages?repo_name=nnstreamer-example

Reply:
Is the pre-built package needed because action artifacts can expire?

Reply:
@niley7464 I mean, if specific permission is required to run the workflow, why don't we just provide a link to the action artifacts or example's repository package?

Reply:
I thought it should be able to check the build progress at certain points in time.
If a pre-built package is okay, I will add action artifacts link.

Reply:
Good to go:)

Reply:
Added documentation that users can download applications via github workflow artifacts.